src/UIState/SeePh.cpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * SeePIDConstants.cpp
+ */
+
+#include "SeePh.h"
+
+#include "Devices/DateTime_TC.h"
+#include "Devices/LiquidCrystal_TC.h"
+#include "Devices/PHControl.h"
+#include "Devices/PHProbe.h"
+#include "TC_util.h"
+
+void SeePh::loop() {
+  int32_t elapsedSeconds = ((millis() - startTime) / 3000) % 2;
+  switch (elapsedSeconds) {
+    case 0:
+      loadHeader(0);
+      loadValues(1);
+      break;
+    case 1:
+      loadPhSetType(0);
+      loadTypeVariables(1);
+      break;
+  }
+}
+
+void SeePh::loadPhSetType(uint16_t line) {
+  char buffer[17];
+  char *type;
+  switch (PHControl::instance()->getPhSetType()) {
+    case FLAT_TYPE: {
+      type = (char *)"flat";
+      break;
+    }
+    case RAMP_TYPE: {
+      type = (char *)"ramp";
+      break;
+    }
+    case SINE_TYPE: {
+      type = (char *)"sine";
+      break;
+    }
+    default:
```

Comment:
While it is unlikely that the default case would ever happen, as long as you have it you could assign a value of "????" to `type` (as it is you have a random value in that variable).